Classes/EventStore/EventTransport.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Cqrs\Event\EventInterface;
+
+/**
+ * Wrapper for a event that was loaded from the Event Store.
+ * It contains the raw event data including version & metadata (storedEvent)
+ * and the converted EventInterface instance (event)
+ */
+final class EventTransport
```

Comment:
> I consider StoredEvent an internal thing of the EventStore, so it shouldn't even land in the EventStream

Right, that's why I struggled with the name.
But it's more than a DomainEventWithMetaData:
It's the raw event before it has been converted, including raw payload, raw metadata, the version in its stream and (with #6) a unique id and a global sequence. All these infos can be important